libs/wait-for-ecs-deploy/README.md

```diff
@@ -0,0 +1,15 @@
+# wait-for-ecs-deploy
+
+Provides a pulumi resource which waits for an ECS deployment to complete. Once the deployment completes, it checks whether the deployment was successful.
+
+If a deployment error is detected, a pulumi error is raised.
+
+## TODO
```

Comment:
If we could fill these out, and add a `## Usage` section

Reply:
Done